framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
If the "execute" event is fired multiple times or only once depends if you attach the listener to the buttons or the command. The idea is to attach it to the command and add that command to multiple buttons. I've modified your playground example a bit to show this: http://tinyurl.com/gna4fkv .